code/game/turfs/closed/wall/reinf_walls.dm

```diff
@@ -215,9 +215,13 @@
 			dismantle_wall()
 
 /turf/closed/wall/r_wall/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
-	if(the_rcd.canRturf || the_rcd.construction_mode == RCD_WALLFRAME)
+	if (the_rcd.construction_mode == RCD_WALLFRAME)
 		return ..()
-
+	if(!the_rcd.canRturf)
+		return
+	. = ..()
+	if (.)
+		.["delay"] *= RCD_RWALL_DELAY_MULT
```

Comment:
This adds delay to the admin RCD & industrial RCD cause they too can deconstruct rwalls. Is this intended? If so you should ad that to your changelog as well

Reply:
Yeah this is intended, done